### DISCUSSION TITLE
Support gitmux.yaml

### DISCUSSION THREAD

**Discussion by joshmedeski** [arl/gitmux#106] [Ideas] [answered]
I've been having issues with my Neovim setup and LSP detecting the gitmux.conf file as I currently associate `*.conf` files has bash files:

```
Failed to attach to yamlls for current buffer. Already attached to bashls
```

Could we allow the `.yaml` or `.yml` extension to be supported since that's the structure of the document?

**Comment by arl** [comment]
Calling `gitmux -cfg FILE` should work , whatever extension FILE has.

**Comment by joshmedeski** [comment]
When I change it to `yml` gitmux doesn't load.

![SCR-20231019-jixp](https://github.com/arl/gitmux/assets/2036594/cbaad0fe-be27-456f-b180-b6a0fea2bb8b)

I made sure to restart tmux completely and as soon as I change it back to `conf` it immediately starts working again.

**Comment by arl** [comment, marked as answer]
Not sure what's happening on your configuration but I can guarantee that `gitmux -cfg WHATEVER` will make gitmux load the file called WHATEVER and decode it as yaml.

See:

![image](https://github.com/arl/gitmux/assets/476650/d57b5d56-8777-463a-8b75-348737c65383)

You should not need to restart tmux, `tmux source tmux.conf' should be enough. But if you do want to be sure to restart, `tmux kill-server` should work.

**Comment by joshmedeski** [comment]
Thanks for clarifying, it was because of my `mackup` configuration which I had to manually reconfigure and it's working!

**Comment by arl** [comment]
Happy to help
